src/AppBundle/Entity/Repository/TeamRepository.php

```diff
@@ -64,4 +64,17 @@ public function findByTeamInterestAndSemester(Semester $semester)
             ->getQuery()
             ->getResult();
     }
+
+    public function findByDepartmentAndName(string $departmentCity, string $name)
```

Comment:
`findByDepartmentCityAndName` eller `findByCityAndName`?

Reply:
City and name er vel best